model/k8s/metrics.yaml

```diff
@@ -469,3 +469,200 @@ groups:
     attributes:
       - ref: k8s.namespace.phase
         requirement_level: required
+
+  # k8s.resourcequota.cpu.* metrics
+  - id: metric.k8s.resourcequota.cpu.limit
+    type: metric
+    metric_name: k8s.resourcequota.cpu.limit
+    stability: development
+    brief: |
+      The CPU limits in a specific namespace.
+      Across all pods in a non-terminal state,
+      the sum of CPU limits cannot exceed this value.
```

Comment:
```suggestion
    brief: |
      The CPU limits in a specific namespace.
      - If `k8s.resourcequota.state` is `hard`: this value is the maximum allowed CPU limit enforced by the quota.
      - If `k8s.resourcequota.state` is `used`: this value is the current usage of CPU limits requested by all non-terminal pods.
```

Reply:
We'll have to add such clarification in each metric :(

What do you think about adding a note in the attribute's definition and referencing that in the metrics?
Example note in attribute
```
  note: |
    The meaning of the metric value depends on the state:
    - If state is `hard`, the value represents the configured quota limit of the resource in the namespace.
    - If state is `used`, the value represents the current observed total usage of the resource in the namespace.
```
And in the metrics, put a line `See `k8s.resourcequota.state` attribute notes (<link>) for further details.`